src/reporting-guide.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: content.liquid
+title: Reporting Guide
+---
+
+### Reporting Process
+
+If you experience any harassment, or unfair treatment, or witness any behavior you believe violates the PyCon India [Code Of Conduct](https://in.pycon.org/2024/code-of-conduct/), we encourage you to report it as soon as possible. Reports can be made through the following channels:
+
+- **Code of Conduct (CoC) Workgroup Area:** You can directly approach the CoC workgroup area with any concerns you may have. If you are unable to locate the CoC workgroup area, please ask one of our volunteers wearing Volunteer badges for assistance. They will gladly direct you to the CoC Workgroup area.
+
+- **Email:** You can send an email to [coc@in.pycon.org](mailto:coc@in.pycon.org) to submit your report electronically.
+
+- **In-person:** If you prefer to discuss the matter in-person, please notify any member of the PyCon India organizing team or the Code Of Conduct Committee.
```

Comment:
I think to write this line, we need to mention somewhere in this page who are the ones in the CoC Committee?

Reply:
Yeah adding the COC Committee members' contact details.